src/core/vdom/patch.js

```diff
@@ -27,6 +25,8 @@ import {
   isPrimitive
 } from '../util/index'
 
+const isTextInputType = makeMap(TEXT_INPUT_TYPES)
```

Comment:
Maybe we could move function `isTextInputType` to [src/platforms/web/util/element.js](https://github.com/vuejs/vue/blob/dev/src/platforms/web/util/element.js).